components/aws/sagemaker/tests/integration_tests/component_tests/test_deploy_component.py

```diff
@@ -101,6 +105,20 @@ def test_create_endpoint(
             ]
             == "InService"
         )
+        if "ExpectedInstanceType" in test_params.keys():
+            new_endpoint_config_name = sagemaker_utils.describe_endpoint(sagemaker_client, input_endpoint_name)[
+                    "EndpointConfigName"
+                    ]
+            response = sagemaker_utils.describe_endpoint_config(sagemaker_client, new_endpoint_config_name)
+            instance_type = ""
+            for prod_variant in response['ProductionVariants']:
```

Comment:
is there any value in iterating ? maybe we can fetch value in first index for now

Reply:
Done